config/jobs/kubernetes/release/OWNERS

```diff
@@ -1,18 +1,10 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
-- calebamiles
-- justaugustus
-- tpepper
+- release-engineering-approvers
```

Comment:
I wish we had a better way of displaying who these members were to people unfamiliar with the system.

Reply:
Agree, I feel like natively supporting GitHub Teams (ref: https://github.com/kubernetes/test-infra/issues/10065) would be ideal, but this is the system we have in place today.